Argument for '--jsx' option must be: 'preserve', 'react-native', 'react'.

**Steps to reproduce:**
1. Create new project with npx create-scandipwa-app package name.
2. Try to override any typescript component in js.
3. See error on compile

**Actual result**
After making override of tsx files in js compiling is broken.
![image](https://user-images.githubusercontent.com/68007919/199917491-60ded82d-6be4-47ad-90cb-7e3bdba44400.png)


**Expected result**
Should compile without this error.

**Versions**
- ScandiPWA: 6.0.1

i am getting this error if i am trying to install by npx create scandipwa-app pwa. it is being successfull afte that second command when i run this BUILD_MODE=magento npm run build i am getting this error (error TS6046: Argument for '--jsx' option must be: 'preserve', 'react-native', 'react'.) can you please fix this issue?. also tell me for latest npx what should be requirement like

node version
redis
varnished
scandipwa
typescript

thank you

--- a/build-packages/csa-generator-theme/template/.eslintrc.js
+++ b/build-packages/csa-generator-theme/template/.eslintrc.js
@@ -6,6 +6,9 @@ module.exports = {
         tsconfigRootDir: __dirname,
         project: './tsconfig.json',
         requireConfigFile: false,
+        babelOptions: {
+            presets: ['@babel/preset-react'],
+        },
     },
     rules: {
         '@typescript-eslint/object-curly-spacing': 'off',

